### ISSUE TITLE
Expose Date picker component to be used standalone

### ISSUE THREAD

**Issue by tratteo** [Vuepic/vue-datepicker#1001]
**Is your feature request related to a problem? Please describe.**
I want to create a responsive DatePicker. Currently the component uses a floating dialog to display the calendar. I want to open a bottom sheet when the screen is small (mobile). Currently I do not think I have access to components used to create the calendar in the popup dialog.

**Describe the solution you'd like**
I would like to have the possibility to access the Calendar component so that I can decide whether to display it in the default popup or in a custom modal sheet. As an example, [this package](https://vcalendar.io/datepicker/basics.html) exposes the VDatePicker component that has no popup and is just a component that renders a calendar and allows selection.

**Additional context**
To sum up, currently the DatePicker component provided takes care of opening the default popup to allow selection, but we do not have access to the component rendered inside this popup to use it standalone.

**Comment by gmenunes** [comment]
I have a similar problem. I want to use the same Month and Year pickers but make slight changes using slots.

Currently, there is no way to do this. If I use the month-year slot I have to rewrite all logic from scratch, which makes no sense.

**Comment by Jasenkoo** [comment]
@tratteo The item you are looking for is `inline` prop

@gmenunes There are different slots for overlay, overlay items, or just month or year buttons, no need to override the whole block

**Comment by gmenunes** [comment]
> @gmenunes There are different slots for overlay, overlay items, or just month or year buttons, no need to override the whole block
You're right, I didn't understand it in this way when I read the docs for the first time.
Thank you!
